code/lib/core-server/src/build-static.ts

```diff
@@ -114,7 +116,7 @@ export async function buildStaticStandalone(options: BuildStaticStandaloneOption
     features,
   };
 
-  if (staticDirs && options.staticDir) {
+  if (options.staticDir && !isEqual(staticDirs, defaultStaticDirs)) {
```

Comment:
If we could do this without lodash I guess it'd be nicer, but if we can't that's fine too!